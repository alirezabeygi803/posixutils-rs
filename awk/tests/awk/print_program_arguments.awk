BEGIN {
    print ARGC
    print ARGV[0]
    print ARGV[1]
    print ARGV[2]
    print ARGV[3]

    ARGV[0] = ""
    ARGV[1] = ""
    ARGV[2] = ""
    ARGV[3] = ""
}