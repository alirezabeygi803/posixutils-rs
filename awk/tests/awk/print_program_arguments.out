4
awk
one
two
three
